swap_meet/clothing.py

```diff
@@ -1,2 +1,11 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    
+    def __init__(self, condition=0.0):
+        """ Constructs all the necessary attributes for the item object. """
+        super().__init__(category="Clothing", condition=condition)
```

Comment:
👍  great job hardcoding the `category` param inside the `super`. That way a user can't accidentally or intentionally change the attribute!